core/src/com/mygdx/minigolf/controller/EntityFactory.java

```diff
@@ -0,0 +1,48 @@
+package com.mygdx.minigolf.controller;
```

Comment:
Er entities del av model? Burde isåfall entityfactory være i model-pakken? 
Er jo en slags logikk/kontroll i å lage entities da, så vet ikke helt. Kan være jeg har misforstått ECS også.

Reply:
@AleksanderWK hva tenker du om dette? Skal vi la den bli i controller-pakken?